_data/simple-icons.json

```diff
@@ -2550,6 +2550,11 @@
             "hex": "000000",
             "source": "https://tidal.com"
         },
+        {
+            "title": "Tik Tok",
+            "hex": "000000",
```

Comment:
The color for this one is not very clear 🤔 In general it's nice to shy away from black as it is a very generic color though this case I don't think it is necessarily wrong.

That said, looking at [the UI in the app](https://play.google.com/store/apps/details?id=com.zhiliaoapp.musically&referrer=af_tranid%3DULC8xI-j1M4RlbXYFLjVPw%26pid%3Dtiktokweb), I think the pink color (`#e01a4e`) may be a bit better as brand color.

Just my two cents, thoughts @as280093 and @kze?

Reply:
> ok i edit the pr as for now i am on my phone and for the color using black is my opinion as background of their official logo is black ⚫ | [in [this comment](https://github.com/simple-icons/simple-icons/pull/1376#issuecomment-486810673)]

Pinging @davidklebanoff for a third opinion

Reply:
Repinging @davidklebanoff, also pinging @birjolaxew now